packages/static-build/test/integration-setup.js

```diff
@@ -38,8 +38,14 @@ module.exports = function setupTests(groupIndex) {
     console.log('testing group', groupIndex, fixtures);
   }
 
+  const fixturesToSkip = ['jekyll-v3', 'jekyll-v4', 'middleman-v4'];
+
   // eslint-disable-next-line no-restricted-syntax
   for (const fixture of fixtures) {
+    if (fixturesToSkip.includes(fixture)) {
```

Comment:
I think we should have included a comment here about why this are skipped and when we plan to restore them.